tt_metal/hw/inc/blackhole/eth_l1_address_map.h

```diff
@@ -56,15 +55,12 @@ struct address_map {
   // erisc early exit functionality re-uses mailboxes_t::ncrisc_halt_msg_t::stack_save memory
   static constexpr std::int32_t ERISC_MEM_MAILBOX_STACK_SAVE = ERISC_MEM_MAILBOX_BASE + 4;
 
-  static constexpr std::int32_t ERISC_RING_BUFFER_ADDR = ERISC_MEM_MAILBOX_BASE + 256 + 16;
-  static constexpr std::int32_t PRINT_BUFFER_ER = ERISC_RING_BUFFER_ADDR + RING_BUFFER_SIZE;
+  static constexpr std::int32_t PRINT_BUFFER_ER = ERISC_MEM_MAILBOX_BASE + 288 + 16;
```

Comment:
288 here?

Reply:
Yeah, all the values for sizes are hard coded but asserted.  I have (at least) 2 more passes to make over this, removing profiler and splitting kernels/firmware.  I'll be sure to remove the magic when the dust settles and before I move onto the next thing